tests/lib/rules/jest-dom-prefer-enabled-disabled.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * @fileoverview prefer toBeDisabled or toBeEnabled over attribute checks
+ * @author Ben Monro
+ */
+'use strict';
+const rule = require('../../../lib/rules/jest-dom-prefer-enabled-disabled');
+
+const RuleTester = require('eslint').RuleTester;
+
+const ruleTester = new RuleTester({
+  parserOptions: { ecmaVersion: 2015, sourceType: 'module' },
+});
+ruleTester.run('jest-dom-prefer-enabled-disabled', rule, {
+  valid: [
+    `expect(element).toBeDisabled()`,
+    `expect(element).toHaveProperty('checked', true)`,
+  ],
+
+  invalid: [
```

Comment:
It would be good to add some tests in case the user is using Testing Library's queries in the `expect` statement:

```js
expect(getByText("submit")).toBeDisabled()
```